database/database_test.go

```diff
@@ -0,0 +1,133 @@
+package database
+
+import (
+	"fmt"
+	"testing"
+
+	obj "github.com/jcgallegdup/Concurrent-Document-Editor/objects"
+)
+
+const path = "../updates.db"
+const invalidID = 9001
+const ID = 1
+
+var expectedChanges = []obj.Change{obj.NewChange("insert", 0, "a"), obj.NewChange("insert", 1, "b")}
```

Comment:
In the spirit of keeping making tests independent of each other (see my other long comment in this file lol), I'd suggest avoiding global variables and declaring variables like these anew inside each test function as you need them.

Although, this is more about style and less of a big deal than having tests tests not depend on each other's behaviour.

Reply:
This one I think I disagree with. Imo it makes it cleaner to not have to re-declare constants inside each test case, especially frequently used ones. Source: they encouraged us to create global constants for tests at my last coop.

Reply:
I think having some global constants that are common to all tests is fine as well. The main thing is just that each test should pass/fail independently of whether another test does.

Reply:
I'm ok with that 👍